providers/gce/gce.go

```diff
@@ -246,6 +248,11 @@ type ConfigGlobal struct {
 	// Default to none.
 	// For example: MyFeatureFlag
 	AlphaFeatures []string `gcfg:"alpha-features"`
+
+	// UnmanageFirewallRules should be set to true when the provider should not
+	// create, delete, or update firewall rules. For instance, when a user does not have the
+	// permissions to create firewall rules the rule creation should be skipped to avoid errors.
+	UnmanageFirewallRules bool `gcfg:"manage-firewall-rules"`
```

Comment:
UnmanageFirewallRules feels a little awkward on the tongue

Perhaps SkipFirewallManagement? Or something where it is an enum, since we prefer those anyway over bools. `FirewallManagement: Managed | Unmanaged // when omitted, Managed`

Reply:
@bowei If we went with `DisableFirewallManagement` do we want to turn that into `EnableFirewallManagement`? I like the Disable for the default setting which would be False. I know it is a double negative but ensures backwards compatibility right ?